airflow/dags/uhc.py

```diff
@@ -0,0 +1,35 @@
+# Ignore the Airflow module, it is installed in both dev and prod
+from airflow import DAG  # type: ignore
+from airflow.utils.dates import days_ago  # type: ignore
+
+import util
+
+_UHC_BASE_URL = "https://www.americashealthrankings.org/api/v1/downloads/210"
+_UHC_WORKFLOW_ID = 'UHC_DATA'
+_UHC_DATASET_NAME = 'uhc_data'
+
+default_args = {
+    'start_date': days_ago(0),
+}
+
+data_ingestion_dag = DAG(
+    'uhc_ingestion_dag',
+    default_args=default_args,
+    schedule_interval='@yearly',
```

Comment:
Do you necessarily want a yearly schedule for this, as opposed to just having it be manually run?